<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>808</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,5">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1070</width>
         <height>243</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Zapis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="text">
             <string>Wczytaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_network">
            <property name="text">
             <string>Sieć</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_status">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Interwał [ms]</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="interwalSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxCzarnyMotyw">
            <property name="text">
             <string>Czarny motyw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="4">
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="GornaGranica">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnModelARx">
              <property name="text">
               <string>ModelARX</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Dolna Granica</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Górna Granica</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="DolnaGranica">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="AntyWindUP">
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Antywindup</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::VLine</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="midLineWidth">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::Shape::VLine</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="midLineWidth">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="okresSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="wypelnienieSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="valueSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Wartość zadana</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblStala">
              <property name="text">
               <string>Składowa stała</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lblWypelnienie">
              <property name="text">
               <string>Wypełnienie</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lblOkres">
              <property name="text">
               <string>Okres</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblAmplituda">
              <property name="text">
               <string>Amplituda</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="skladowaStalaSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="czasAktywacjiSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblCzasAktywacji">
              <property name="text">
               <string>Czas aktywacji [ms]</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Td:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>Stała pod sumę</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QDoubleSpinBox" name="tdSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.200000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="kSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="tiSpinBox">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Stała przed sumę</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Ti:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>P:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Regulator PID</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1084</width>
         <height>542</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="1" column="1">
         <widget class="QCustomPlot" name="nastawyPlot" native="true"/>
        </item>
        <item row="1" column="2">
         <widget class="QCustomPlot" name="uchybPlot" native="true"/>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QCustomPlot" name="zadajnikPlot" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QCustomPlot" name="regPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
